samples/javascript_nodejs/05.multi-turn-prompt/dialogs/userProfileDialog.js

```diff
@@ -98,15 +104,32 @@ class UserProfileDialog extends ComponentDialog {
         }
     }
 
-    async confirmStep(step) {
+    async pictureStep(step) {
         step.values.age = step.result;
 
         const msg = step.values.age === -1 ? 'No age given.' : `I have your age as ${ step.values.age }.`;
 
         // We can send messages to the user at any point in the WaterfallStep.
         await step.context.sendActivity(msg);
 
-        // WaterfallStep always finishes with the end of the Waterfall or with another dialog, here it is a Prompt Dialog.
+        if (step.context.activity.channelId === channels.msteams) {
+            // This attachment prompt example is not designed to work for Teams attachments, so skip it in this case
+            return await step.next(undefined);
```

Comment:
Might be beneficial to directly notify the user that this won't work in Teams

```suggestion
            await step.context.sendActivity('This sample does not work in the Teams channel. Please test this in a different channel');
            return await step.next(undefined);
```

Reply:
@mdrichardson - The sample works fine in Teams, though. It just won't have the attachment prompt in it.

Reply:
@v-kydela Right. Maybe a different message, then. I think a common way of testing samples is to download it and run it without reading the code. If somebody happens to do that in Teams, they might be confused. They might benefit from a notification for it. Consider this a nit-picky comment